clients/lsp-csharp.el

```diff
@@ -235,6 +241,99 @@ tarball or a zip file (based on a current platform) to TARGET-DIR."
   "Resolves path and arguments to use to start the server."
   (list (lsp-csharp--language-server-path) "-lsp"))
 
+(defun lsp-csharp--reset-test-buffer (present-buffer)
+  "Create new or reuse an existing test result output buffer.
+
+PRESENT-BUFFER will make the buffer be presented to the user."
+  (let ((existing-buffer (get-buffer lsp-csharp-test-run-buffer-name))
+        (project-root-dir (lsp--suggest-project-root)))
+    (if existing-buffer
+        (progn
+          (with-current-buffer existing-buffer
+            (setq buffer-read-only nil)
```

Comment:
usually, this is performed in `let` bound `buffer-read-only`

Reply:
This would be `((inhibit-read-only t))`, no?